{"name": "test_count_endpoint[daily-kolonka-bypass]", "status": "broken", "statusDetails": {"message": "TypeError: 'int' object is not callable", "trace": "api_context = <playwright._impl._fetch.APIRequestContext object at 0x000002117FD9D010>, status = 'bypass', tag = 'kolonka', date_type = 'daily'\n\n    @pytest.mark.parametrize(\"status\", STATUSES)\n    @pytest.mark.parametrize(\"tag\", TAGS)\n    @pytest.mark.parametrize(\"date_type\", DATE_TYPES)\n    def test_count_endpoint(api_context, status, tag, date_type):\n        count_api = StatisticsAPI(api_context)\n        params = {\n            **DEFAULT_PARAMS,\n            \"status\": status,\n            \"tag\": tag,\n            \"date_type\": date_type\n        }\n        response = count_api.get_counts(**params)\n>       assert response.status == 200 (\n            f\"Unexpected status code: {response.status} for \"\n            f\"status={status}, tag={tag}, date_type={date_type}\"\n        )\nE       TypeError: 'int' object is not callable\n\ntest_products.py:171: TypeError"}, "parameters": [{"name": "date_type", "value": "'daily'"}, {"name": "tag", "value": "'kolonka'"}, {"name": "status", "value": "'bypass'"}], "start": 1759751427306, "stop": 1759751427371, "uuid": "66e3cf76-f00f-4d8f-9e1b-6879500716c4", "historyId": "d2fe7a5177382ae139bae0120345158d", "testCaseId": "a1a25b2e255c1aeacc61b88f5be73e55", "fullName": "tests.test_products#test_count_endpoint", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_products"}, {"name": "host", "value": "PC-2993"}, {"name": "thread", "value": "23396-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_products"}], "titlePath": ["tests", "test_products.py"]}